src/PhpSpreadsheet/Calculation/Web.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace PhpOffice\PhpSpreadsheet\Calculation;
+
+use GuzzleHttp\Client;
+use GuzzleHttp\Exception\ClientException;
+use GuzzleHttp\Exception\ConnectException;
+
+class Web
+{
+    protected static $client;
+
+    /**
+     * Override the default webclient.
+     * Used for test purposes.
+     *
+     * @param Client $client
+     */
+    public static function setClient(Client $client)
+    {
+        self::$client = $client;
+    }
+
+    /**
+     * WEBSERVICE.
+     *
+     * Returns data from a web service on the Internet or Intranet.
+     *
+     * Excel Function:
+     *        Webservice(url)
+     *
+     * @category Web Functions
+     *
+     * @param string $url
+     *
+     * @return string the output resulting from a call to the webservice
+     */
+    public static function WEBSERVICE($url)
```

Comment:
Use proper typing for all new code

```suggestion
    public static function WEBSERVICE(string $url)
```